Terminal.Gui/Views/TextView.cs

```diff
@@ -78,6 +84,27 @@ public static List<List<Rune>> StringToRunes (ustring content)
 			return lines;
 		}
 
+		// Splits a string into a List that contains a List<Rune> for each line
+		public static List<List<Rune>> StringToRunes (ustring content, int nMaxRuneCount)
```

Comment:
Ok, I see what is going on here.   The parameter is not really the maxRuneCount, but the desired column. 

This should be updated accordingly.